mappings/net/minecraft/item/OnAStickItem.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_1758 net/minecraft/item/OnAStickItem
+	FIELD field_23253 target Lnet/minecraft/class_1299;
+	METHOD <init> (Lnet/minecraft/class_1792$class_1793;Lnet/minecraft/class_1299;)V
+		ARG 1 settings
+		ARG 2 target
+	METHOD method_20271 (Lnet/minecraft/class_1268;Lnet/minecraft/class_1657;)V
+		ARG 1 p
```

Comment:
`player`?

Reply:
Yes, but it's a lambda that was already there, idk if we should map them

Reply:
Should prob leave this, guess otherwise there is chance for name clash